olive/olive_config.json

```diff
@@ -0,0 +1,234 @@
+{
+    "passes": {
+        "AppendPrePostProcessingOps": {
+            "module_path": "olive.passes.onnx.append_pre_post_processing_ops.AppendPrePostProcessingOps"
```

Comment:
Not related to this PR:
This central passes config looks nice. If we think further, it might be a good place to put pass's attributes like:
1. which ep this pass can be run
2. which precision(float32/float16/int4) the passes can conduct.
3. which ort version it is supported
That might be helpful for auto optimization as well.
Like this kind of configs
https://github.com/microsoft/Olive/blob/main/olive/auto_optimizer/config_template/pass_capability.yaml

Reply:
>  If we think further, it might be a good place to put pass's attributes

Agreed. There is lot more that can be data driven allowing easier user configuration changes.